aws/table_aws_redshift_cluster.go

```diff
@@ -33,6 +33,13 @@ func tableAwsRedshiftCluster(_ context.Context) *plugin.Table {
 				Description: "The unique identifier of the cluster.",
 				Type:        proto.ColumnType_STRING,
 			},
+			{
+				Name:        "arn",
```

Comment:
@anisadas Since we're adding this column, can you please remove the `cluster_namespace_arn` column? We should also be able to use a `transform.fromField("ClusterNamespaceArn")` transformation then instead of having a separate function for it.